simplesmtp/index.d.ts

```diff
@@ -0,0 +1,155 @@
+﻿// Type definitions for simplesmtp 0.3.35
+// Project: https://github.com/andris9/simplesmtp
+// Definitions by: Rogier Schouten <https://github.com/rogierschouten/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module "simplesmtp" {
```

Comment:
do not use the external module declaration, just remove `declare module "simplesmtp" {` and everything should still work.